README.md

```diff
@@ -51,16 +48,35 @@ Add the following configuration to your project:
         def project do
             [
                 ...
-                grisp: [
-                    otp: [verson: "21.0"],
-                    deploy: [
-                        # pre_script: "rm -rf /Volumes/GRISP/*",
-                        # destination: "/Volumes/GRISP",
-                        # post_script: "diskutil unmount /Volumes/GRISP",
-                    ]
-                ],
+                grisp: grisp(),
+                releases: releases()
+            ]
+        end
+
+        def grisp do
+            [
+                otp: [verson: "26.0.2"],
```

Comment:
Better just "26" so it would pick the highest OTP pakcage version we have available for 26.